src/status_im/contact/core.cljs

```diff
@@ -81,6 +81,14 @@
                 (send-contact-request contact)
                 (mailserver/process-next-messages-request)))))
 
+(fx/defn remove-contact
+  "Remove a contact from current account's contact list"
+  {:events [:contact.ui/remove-contact-pressed]}
+  [{:keys [db] :as cofx} public-key]
+  (fx/merge cofx
+            {:db            (update db :contacts/contacts dissoc public-key)
+             :data-store/tx [(contacts-store/delete-contact-tx public-key)]}))
```

Comment:
i believe we don't use `:data-store/tx` for contacts anymore @cammellos could you assist, please?

Reply:
for sure we don't, the question is do we have a "shhext_deleteContact" rpc method to replace it? @cammellos 

if we don't I'd say just add a TODO here that says the contact is not actually remove, because this is a UI PR and there is no point blocking it waiting for a core feature. Remove that :data-store/tx it the delete-contact-tx function won't exist once you rebase anyway.

Reply:
i added a TODO

Reply:
should we remove the contact completely (needs a new endpoint in status-go), or is it enough to remove the `contacts/added` tag (does not need a new endpoint)? 

If we delete it completely, it will come back (maybe not in the list if we are filtering by `contact/added` tag), next contact update, so not sure how useful it is, but I don't have much context on this task.

Reply:
I suppose we just want to remove contacts/added, it removes the user from the contact list right?

Reply:
Not sure about that, I would guess so as it should only show contacts that we added.

Essentially we can just remove the `contact/added` tag and call `save-contact` in `status-im.data-store.contacts`

Reply:
@cammellos @yenda take a look to the latest push